lib/contentProfileListener.js

```diff
@@ -98,16 +98,36 @@ class ContentProfileImporter extends SHRContentProfileParserListener {
 
     if (currentElement) {
       for (const name of names) {
+        let parentElements = this.getRecursiveBasedOns(currentElement.identifier);
+        
         if (name === 'Value' || name === '_Value') {
           path.push(new Identifier('', '_Value'));
 
-          if (currentElement.value && !(currentElement.value instanceof ChoiceValue)) {
+          // find value from self or most recent ancestor
+          let value = null;
+          for (const id of parentElements) {
+            const el = this._specs.dataElements.findByIdentifier(id);
+            value = el ? el.value : null;
+
+            if (value) {
+              break;
+            }
+          }
+
+          if (value && !(value instanceof ChoiceValue)) {
             currentElement = this._specs.dataElements.findByIdentifier(currentElement.value.effectiveIdentifier);
```

Comment:
Shouldn't this be using the new `value` variable now?  (not `currentElement.value`)